udata/core/site/models.py

```diff
@@ -23,3 +28,30 @@ class Site(WithMetrics, db.Document):
     configs = db.DictField()
     themes = db.DictField()
     settings = db.EmbeddedDocumentField(SiteSettings, default=SiteSettings)
+
+
+def get_current_site():
```

Comment:
Because of:
- coherence: `current_site` only refers to the Site model
- circular dependencies: avoid models to import from views

Nothing changed beside the location.
The related import statements are all changed into this PR and test are passing.
Given the dependency we have on it, they would have failed if there was an issue.